packages/imperative/src/config/src/ProfileInfo.ts

```diff
@@ -1225,6 +1234,312 @@ export class ProfileInfo {
         LoggerUtils.setProfileSchemas(this.mProfileSchemaCache);
     }
 
+    /**
+     * Reads the `extenders.json` file from the CLI home directory.
+     * Called once in `readProfilesFromDisk` and cached to minimize I/O operations.
+     * @internal
+     */
+    public static readExtendersJsonFromDisk(): IExtendersJsonOpts {
+        const extenderJsonPath = path.join(ImperativeConfig.instance.cliHome, "extenders.json");
```

Comment:
Should we document (maybe with JSDoc) how the `extender.json` file looks like?

We don't have to be too verbose, but something like the Pseudocode examples in this file might help 
- https://github.com/zowe/zowe-cli/blob/master/packages/imperative/src/config/src/ProfileInfo.ts#L61

Reply:
Yes, thanks for catching that - ideally, I'd like to keep knowledge of that structure to developers only, especially since we don't intend to expose this file to users. I think this would fit well in the typedoc somewhere.

I was thinking that I should add that comment above the `IExtenderJsonOpts` type in the `IExtenderOpts.ts` file, especially since I am storing related types in that area 😋

Reply:
Added in [4b72f50](https://github.com/zowe/zowe-cli/pull/2001/commits/4b72f50deeed88c3debf557a0158af890ab0e7a8)